### ISSUE TITLE
Better Default Colour for Subtitles

### ISSUE THREAD

**Issue by DarioS** [garthtarr/sydney_xaringan#1]
The default colour seems to be the same as the background, which makes the text appear to be invisible. For example, I have

```
title: "Classification for High-dimensional Biomedical Data"
subtitle: "Utilising ClassifyR"
```

to break the long title into two lines. But, the subtitle can only be seen if dragged across with the mouse cursor.

![image](https://user-images.githubusercontent.com/631218/47972223-4b99a980-e0ee-11e8-936e-2354eca40941.png)

I also tried

```
title: |
       | Classification for High-dimensional Biomedical Data
       | Utilising ClassifyR
```

but the output was not as expected (colour and size of second line had changed).

![image](https://user-images.githubusercontent.com/631218/47972281-e09ca280-e0ee-11e8-82bc-3c60d8014550.png)

**Comment by garthtarr** [comment]
Fixed with a09520242b35973f613e87efb896863c0a9e8dc4

**Comment by garthtarr** [comment]
As background, this was the cuplrit:
```
.remark-slide-content > h2:first-of-type {
  color: #e64626;
  margin-top: 5px;
  margin-bottom: 10px;
}
```
Which was done to make the h2 titles on each slide match powerpoint. But it also did it on the first slide, so I've used `important!` on the first slide (ugly css, I know).
```
.title-slide h2 {
  color: black !important; /* first of type is master brand red */
  font-size: 35px;
  padding-left: 20px;
  padding-bottom: 10px;
  margin-top: 0px;
  text-align: left;
  vertical-align: middle;
}
```

**Comment by garthtarr** [comment]
For your use case though, it's not really a subtitle, did you try using <br> to add a line break? @DarioS

**Comment by garthtarr** [comment]
I just did, fails big time, I wonder why!?!  It's something I've done, because it's not a problem for standard xaringan.

**Comment by garthtarr** [comment]
Ah, a bad `line-height: 0.05;` setting.

**Comment by garthtarr** [comment]
Fixed with 5aa2c475e99bc8ab496ba296d094a609cef59fff
